guacamole/src/main/webapp/app/manage/controllers/manageUserController.js

```diff
@@ -918,6 +961,28 @@ angular.module('manage').controller('manageUserController', ['$scope', '$injecto
             else
                 removeConnectionGroupPermission(identifier);
 
+        },
+
+        /**
+         * Notifies the controller that a change has been made to the given
+         * sharing profile permission for the user being edited. This only
+         * applies to READ permissions.
+         *
+         * @param {String} identifier
+         *     The identifier of the sharing profile affected by the changed
+         *     permission.
+         */
+        sharingProfilePermissionChanged : function sharingProfilePermissionChanged(identifier) {
+
+            // Determine current permission setting
+            var value = $scope.permissionFlags.sharingProfilePermissions.READ[identifier];
```

Comment:
Yes, mostly. It will be true if the permission is granted, and false or undefined otherwise. The `PermissionFlagSet` definition states (copied from the diff of this PR):

>
> ```
 +         * The granted state of each permission for each sharing profile, as a
 +         * map of object permission type string to permission map. The
 +         * permission map is, in turn, a map of sharing profile identifier to
 +         * boolean value. A particular permission is granted if its
 +         * corresponding boolean value is set to true. Valid permission type
 +         * strings are defined within PermissionSet.ObjectPermissionType.
 +         * Permissions which are not granted may be set to false, but this is
 +         * not required.
> ```
>

How would you feel about `granted` instead?